pages/archive.js

```diff
@@ -13,17 +20,21 @@ export default function ArchivesPage({ cards, collections, deleteCollection }) {
 
   return (
     <>
-      <StyledHeadlines>Das kann ich schon!</StyledHeadlines>
+      <GlobalStyle $isArchive />
+      <StyledHeadlines>Dein Archiv</StyledHeadlines>
       {!filteredCards.length ? (
         <StyledText>Dein Archiv ist noch leer.</StyledText>
       ) : (
         <CollectionList
           collections={collections}
           cards={filteredCards}
           deleteCollection={deleteCollection}
-          isMastered={true}
+          archive={true}
```

Comment:
Any value that's not `false`, `0`, `""`, `null`, `undefined`, or `NaN` is considered `truthy` ✨. This means when you pass `archive={true}` as a prop, you're explicitly setting the archive prop to a truthy value. 

```suggestion
          archive
```

Reply:
I thought this explicit writing makes the code more readable for persons, who code with different languages or less experience... but I changed now